homeworks/maksym.selenkov_MaksymSelenkov/6-typescript/src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.bubble {
+  margin: 80px;
+}
+
+.bubble {
```

Comment:
Could you help me to understand why there are two classes with the same selector? If one of them is extra, let's delete it :)